homeassistant/components/somfy_mylink/config_flow.py

```diff
@@ -115,7 +115,23 @@ def __init__(self, config_entry: config_entries.ConfigEntry):
         """Initialize options flow."""
         self.config_entry = config_entry
         self.options = config_entry.options.copy()
-        self._entity_id = None
+        self._target_id = None
+
+    @callback
+    def _async_callback_targets(self):
+        """Return the list of targets."""
+        return self.hass.data[DOMAIN][self.config_entry.entry_id][MYLINK_STATUS][
+            "result"
+        ]
+
+    @callback
+    def _async_get_target_name(self, target_id) -> None:
```

Comment:
Looks like the return value typing is incorrect.
```suggestion
    def _async_get_target_name(self, target_id) -> str:
```